h1/src/lib.rs

```diff
@@ -137,4 +137,29 @@ pub unsafe fn init() {
     cortexm3::nvic::disable_all();
     cortexm3::nvic::clear_all_pending();
     cortexm3::nvic::enable_all();
+
+    // -------------------------------------------------------------------
+
+    // Test code. Only enable if you actually want to run these tests,
+    // since they cause HW wear.
```

Comment:
nvcounter_test and flash_test cause hardware wear, the others are just slow.

Reply:
Ah, thanks for clarifying. I updated the comment.